ansible/redpanda/cluster/docs/README.md

```diff
@@ -0,0 +1,193 @@
+# Ansible Collection for Redpanda
+
+Redpanda Ansible Collection that enables provisioning and managing a [Redpanda](https://www.redpanda.com/) cluster.
+
+## Usage
+
+### Required Steps: Deploying Redpanda
+
+> Note: This playbook is designed to be run repeatedly against a running cluster however be aware that it will cause a
+> rolling restart of the cluster to occur. If you do not want the cluster to restart you can
+> specify `restart_node=false`
+> either globally (as --extra-vars, or on a host-by-host basis in the `hosts.ini` file). If you make changes to the node
+> config and do not perform a restart then you may leave your `rpk` config (and this may inhibit subsequent executions
+> of
+> the playbook). If you re-run the playbook it will overwrite any configurations with those specified in the playbook,
+> so
+> care should be taken to ensure that the playbook contains any desired configuration (for example, if you have enabled
+> TLS on your cluster, any subsequent runs of `provision-node` should be made with `-e tls=true` otherwise the playbook
+> will disable TLS).
+
+Before running these steps, verify that the `hosts.ini` file contains the correct information for your infrastructure.
+This will be automatically populated if using the terraform steps above.
```

Comment:
Same comment as the other Readme.

is the double readme required for galaxy? because in the galaxy.yaml I see a:
```
# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
readme: README.md
```